package.json

```diff
@@ -6,28 +6,41 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview",
-    "prepare": "husky install"
+    "preview": "vite preview"
   },
   "dependencies": {
+    "atomic-router": "^0.8.0",
+    "atomic-router-react": "^0.8.0",
+    "classnames": "^2.3.2",
     "commitizen": "^4.2.5",
     "@ant-design/icons": "^4.8.0",
     "antd": "^5.0.2",
     "effector": "^22.4.0",
     "effector-react": "^22.3.4",
+    "effector-forms": "^0.0.24",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "history": "^5.3.0",
+    "lodash": "^4.17.21",
```

Comment:
there is no point in using Lodash, everything can be done using es5 or another architecture